commons/passive/dreem/dreem_hypnogram.avsc

```diff
@@ -0,0 +1,15 @@
+{
+  "type": "record",
+  "namespace": "org.radarcns.passive.dreem",
+  "name": "Dreem",
```

Comment:
Perhaps the name should be `DreemSleepStage` and the file dreem_sleep_stage.avsc.